v8env/ts/fly/image.ts

```diff
@@ -146,6 +146,18 @@ export class Image {
     i.info = result.info
     return i
   }
+
+  async toResponse(res:any = { headers: { "Content-Type": "image/jpg" }}): Promise<any> {
+    const result = await imageToBuffer(this._ref)
```

Comment:
This should just call `this.toBuffer()` instead of reimplementing the logic here.

Reply:
I will change that.